pkg/controllers/applicationset_controller.go

```diff
@@ -45,10 +46,16 @@ func (r *ApplicationSetReconciler) Reconcile(req ctrl.Request) (ctrl.Result, err
 	var applicationSetInfo argoprojiov1alpha1.ApplicationSet
 	if err := r.Get(ctx, req.NamespacedName, &applicationSetInfo); err != nil {
 		log.Info("Unable to fetch applicationSetInfo %v", err)
-
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	var generator generators.Generator
+	generator = generators.NewListGenerator()
+	for _, tmpGenerator := range applicationSetInfo.Spec.Generators {
+		newApplications, err := generator.GenerateApplications(&tmpGenerator, &applicationSetInfo)
+		log.Infof("newApplications %+v error %++v", newApplications, err)
```

Comment:
Would change this one as well to if err != nil { log.WithError(err).Error("error generating applications")

Reply:
done